docs/dns-usage.rst

```diff
@@ -0,0 +1,191 @@
+Using the API
+=============
+
+Client
```

Comment:
Dunno -- maybe it just needs to note that the other API clients are all similar (actually, `datastore` is the oddball, compared to `storage`/`pubsub`/`bigquery`/`dns`).